lib/services/local/presence.go

```diff
@@ -138,6 +139,112 @@ func (s *PresenceService) DeleteNamespace(namespace string) error {
 	return trace.Wrap(err)
 }
 
+// GetServerInfos returns a stream of ServerInfos.
+func (s *PresenceService) GetServerInfos(ctx context.Context, pageSize int) stream.Stream[types.ServerInfo] {
```

Comment:
Why is `pageSize` a parameter to this, if it's actually not going to change anything for the caller?